src/wp-includes/class-wp-theme-json.php

```diff
@@ -2879,6 +2879,20 @@ public static function remove_insecure_properties( $theme_json ) {
 			if ( ! empty( $output ) ) {
 				_wp_array_set( $sanitized, $metadata['path'], $output );
 			}
+
```

Comment:
Thanks for getting this PR up!

In the doc block for `remove_insecure_properties()` it'd be good to have a `@since` annotation, e.g., something like

`* @since 6.3.1 Preserves global styles block variations when securing styles.`